src/js/components/BarraTotal/index.js

```diff
@@ -1,107 +1,122 @@
 import axios from 'axios';
 
-import * as Util from '../Util'
-import Cartao from '../Cartao'
-
-export default class Barra extends Component {
+import * as Util from '../Util';
+import Cartao from '../Cartao';
 
+export default class BarraTotal extends Component {
     constructor(props) {
         super(props);
 
         this.state = {
-          labels: [],
-          valores: [],
-          orgaos: props.orgaos,
-          mesAnoInicial : props.mesAnoInicial,
-          mesAnoFinal : props.mesAnoFinal,
-          totalAcumulado : 0
+            labels: [],
+            valores: [],
+            orgaos: props.orgaos,
+            mesAnoInicial: props.mesAnoInicial,
+            mesAnoFinal: props.mesAnoFinal,
+            totalAcumulado: 0,
         };
 
-        this.obtemDados = this.obtemDados.bind(this);        
+        this.obtemDados = this.obtemDados.bind(this);
     }
 
-    componentWillReceiveProps(props) {
-      
-      this.setState({ 
-        mesAnoInicial: props.mesAnoInicial,
-        mesAnoFinal : props.mesAnoFinal,
-        orgaos : props.orgaos
-      })
-
-      this.obtemDados(props.mesAnoInicial, props.mesAnoFinal, props.orgaos);
+    componentDidMount() {
+        this.obtemDados(
+            this.state.mesAnoInicial,
+            this.state.mesAnoFinal,
+            this.state.orgaos
+        );
     }
 
-    componentDidMount() {
+    componentWillReceiveProps(props) {
+        this.setState({
+            mesAnoInicial: props.mesAnoInicial,
+            mesAnoFinal: props.mesAnoFinal,
+            orgaos: props.orgaos,
+        });
 
-      this.obtemDados(this.state.mesAnoInicial, this.state.mesAnoFinal, this.state.orgaos);        
+        this.obtemDados(props.mesAnoInicial, props.mesAnoFinal, props.orgaos);
     }
 
-    obtemDados(mesAnoInicial, mesAnoFinal, orgaos){
-      
-      var parametrosDeOrgaosFormatados = Util.preparaParametrosDeOrgaos(orgaos); 
-      axios.get(`https://api.inlinesoft.com.br/taok-api/inicial/ano/${mesAnoInicial}/${mesAnoFinal}?${parametrosDeOrgaosFormatados}`).then(res => {
-          
-          this.calculaAcumulado(res.data[0].valores);
+    obtemDados(mesAnoInicial, mesAnoFinal, orgaos) {
+        const parametrosDeOrgaosFormatados = Util.preparaParametrosDeOrgaos(
+            orgaos
+        );
+        axios
+            .get(
+                `https://api.inlinesoft.com.br/taok-api/inicial/ano/${mesAnoInicial}/${mesAnoFinal}?${parametrosDeOrgaosFormatados}`
+            )
+            .then(res => {
+                this.calculaAcumulado(res.data[0].valores);
 
-          this.setState({
-            labels: res.data[0].labels,
-            valores: res.data[0].valores
-          });  
-        });
+                this.setState({
+                    labels: res.data[0].labels,
+                    valores: res.data[0].valores,
+                });
+            });
     }
 
-    calculaAcumulado(valores){
+    calculaAcumulado(valores) {
+        // const total = 0.0;
```

Comment:
@samuelsilvadev you can remove this comments

Reply:
sure, nice catch!!